### ISSUE TITLE
FS19?

### ISSUE THREAD

**Issue by Jayshua83** [jDanek/better-minimap#9]
Will you be making this available for FS19?

Please?

**Comment by jDanek** [comment]
Yes, I plan to adapt the minimap for FS19 after the new year. If the new engine permits.

**Comment by xCrazyDogx** [comment]
whoopy that sounds cool, i really miss it and hope you will be able to make it work in fs19

**Comment by evilixx** [comment]
> Yes, I plan to adapt the minimap for FS19 after the new year. If the new engine permits.

Hi jDanek, you working on it and make this available for LS19?

**Comment by PDXJedi** [comment]
Did work on this get forgotten/abandoned? FS19 still desperately needs this.

**Comment by jDanek** [comment]
Sorry I'm not currently working on the map, I just don't have enough time to play, let alone programming.

**Comment by Lazersgm** [comment]
shame. i would love to get a fs19 version. i tried just moding the xml but got "Error: D:/My Documents/My Games/FarmingSimulator2019/mods/FS19_BetterMinimap/src/register.lua:7: attempt to index field 'globalI18N' (a nil value)" in the log. as i don't know how to mod i am at a loss.

**Comment by BattleHook** [comment]
Really would love to see this mod converted for FS25......
